json_schema/system/file_descriptor.json

```
@@ -0,0 +1,113 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Checksums and other information for asserting the integrity of a data file.",
+  "additionalProperties": false,
+  "required": [
+    "describedBy",
+    "schema_type",
+    "content_type",
+    "size",
+    "sha256",
+    "crc32c",
+    "file_id",
+    "file_version",
+    "file_name"
+  ],
+  "title": "File descriptor",
+  "name": "file_descriptor",
+  "type": "object",
+  "properties": {
+    "describedBy": {
+      "description": "The URL reference to the schema.",
+      "type": "string",
+      "pattern" : "^(http|https)://schema\\.(.*?)humancellatlas\\.org/system/(([0-9]{1,}\\.[0-9]{1,}\\.[0-9]{1,})|([a-zA-Z]*?))/file_descriptor$"
```

```suggestion
      "pattern" : "^(http|https)://schema(\\.[a-z]+)?\\.humancellatlas\\.org/system/(([0-9]{1,}\\.[0-9]{1,}\\.[0-9]{1,})|([a-zA-Z]*?))/file_descriptor$"
```

- is a bit stricter
- `.*?` is the same as `.*`
- `.*` in general should be avoided in URL patterns since it can be very easily exploited to point to a completely different host. Think `http://schema.badhost.io/#.humancellatlas.org/system/1.2.3/file_descriptor`